### ISSUE TITLE
Finalize mapping from moodid to circumplex affects

### ISSUE THREAD

**Issue by coryschillaci** [berkeley-dsc/destress#19]

**Comment by coryschillaci** [comment]
@anasrferreira and @peparedes Should also upload their manual responses, then we need a strategy to merge with the Mech Turk data.

**Comment by peparedes** [comment]
I can also get more mturk data (re -  law of large numbers :)

Let's merge what we have...  My personal assessment is also already very
biased by the mturk data analysis.

P

**Comment by coryschillaci** [comment]
More turk data won't hurt if there's money to pay for it! If you do run, can you put a test that they correctly identify most of the moodids that occur as circumplex affects as well in order to get credit? The ones that occur in both lists are:  
- happy
- excited
- nervous
- stressed
- sad
- depressed
- bored
- calm
- relaxed
- content (contented in circumplex)

**Comment by coryschillaci** [comment]
I checked the idea of scoring the MT responders by percent of the above list that they assigned correctly to the identical circumplex affect. The majority vote gets 100%, which is good. It looks like the first five do much better than the last five.  
![screen shot 2015-03-11 at 9 56 53 pm](https://cloud.githubusercontent.com/assets/717954/6612449/937aa58c-c839-11e4-81bb-5e96ac6dd354.png)

**Comment by peparedes** [comment]
yeah, so we need more day data... i will not post then the job till tomorrow

thanks

p
